### PR TITLE
feat(select-card)

### PR DESCRIPTION
**Pull request by PaoloTK**
## Description

This PR adds a select card to control select and input_select entities.

select-light

## Related Issue

https://github.com/piitaya/lovelace-mushroom/issues/873

This PR fixes or closes issue: fixes 873

## Motivation and Context

SSIA

## How Has This Been Tested

Demo instance and own instance.

## Types of changes

-   [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
-   [x] 🚀 New feature (non-breaking change which adds functionality)
-   [ ] 🌎 Translation (addition or update a translation)
-   [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

-   [x] My code follows the code style of this project.
-   [x] My change requires a change to the documentation.
-   [x] I have updated the documentation accordingly.
-   [x] I have tested the change locally.
-   [ ] I followed [the steps](https://github.com/piitaya/lovelace-mushroom#maintainer-steps-to-add-a-new-language) if I add a new language .

### CODE REVIEW

**Comment by briis** [comment]
I would love this card. This is really missed in the different entity cards.

**Comment by piitaya** [comment]
What do you think about coloring the select too?
CleanShot 2023-01-10 at 10 25 11

File: src/cards/select-card/select-card.ts [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                    >
+                        ${picture ? this.renderPicture(picture) : this.renderIcon(entity, icon)}
+                        ${this.renderBadge(entity)}
+                        ${this.renderStateInfo(entity, appearance, name)};
+                    </mushroom-state-item>
+                    <div class="actions" ?rtl=${rtl}>
+                        <mushroom-select-option-control
+                        .hass=${this.hass}
+                        .entity=${entity}
+                    ></mushroom-select-option-control>
```
Comment (piitaya) [inline src/cards/select-card/select-card.ts:114]:
```suggestion
                    <div class="actions" ?rtl=${rtl}>
                        <mushroom-select-option-control
                        .hass=${this.hass}
                        .entity=${entity}
                    ></mushroom-select-option-control>
                    </div>             
```
One missing `div`

**Review by piitaya — APPROVED** [review]

**Comment by PaoloTK** [comment]
> What do you think about coloring the select too? CleanShot 2023-01-10 at 10 25 11

Cool idea, should it be a toggle or fixed?

**Comment by piitaya** [comment]
I have the code locally so I can push it. For lights, the slider is colored with the icon color so we can have the same usage here.

**Comment by PaoloTK** [comment]
> I have the code locally so I can push it. For lights and fan, the slider is colored with the icon color so we can have the same usage here.

Sounds good to me!

**Pull request merged**